www/js/diary/infinite_scroll_list.js

```diff
@@ -543,4 +585,6 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
         }
       });
     });
+
+    console.log("Display trips from Infinite Scroll Ctrl ", $scope.data.displayTrips);
```

Comment:
comment out or remove log spew

Reply:
This was removed

_**Confirmed this change as complete**_